spec/statements.md

```diff
@@ -1226,19 +1444,25 @@ The finally block of a try statement is reachable if the try statement is reacha
 
 The end point of a try statement is reachable if both of the following are true:
 
--   The end point of the try block is reachable or the end point of at least one catch block is reachable.
+- The end point of the try block is reachable or the end point of at least one catch block is reachable.
 
--   If a finally block is present, the end point of the finally block is reachable.
+- If a finally block is present, the end point of the finally block is reachable.
 
 ## The checked and unchecked statements
 
 The checked and unchecked statements are used to control the ***overflow-checking context*** for integral-type arithmetic operations and conversions.
 
-[]{#Grammar_checked_statement .anchor}checked-statement:\
-*checked* block
+[#Grammar_checked_statement .anchor](#Grammar_checked_statement .anchor)checked-statement:\
```

Comment:
There should be a single ANTLR block starting on this line and extending to line 1465